ArjunaJTA/cdi/tests/classes/com/hp/mwtests/ts/jta/cdi/transactional/stereotype/extension/AddTransactionalAnnotationExtension.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2017, Red Hat Middleware LLC, and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package com.hp.mwtests.ts.jta.cdi.transactional.stereotype.extension;
+
+import java.util.Arrays;
+import java.util.List;
+
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.spi.Extension;
+import javax.enterprise.inject.spi.ProcessAnnotatedType;
+import javax.transaction.Transactional;
+
+
+/**
+ * Extension adding the {@link Transactional} at any processed bean.
+ */
+public class AddTransactionalAnnotationExtension implements Extension {
+
+    private List<Class<?>> classToProcess = Arrays.asList(new Class[] {NoAnnotationBean.class});
+
+    void processAnnotatedType(@Observes ProcessAnnotatedType<NoAnnotationBean> bean) {
+        if(classToProcess.contains(bean.getAnnotatedType().getJavaClass())) {
```

Comment:
This `if` and the list above can be removed completely.

Reply:
thanks, I omitted to remove that part of the code